.changeset/poor-schools-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/canon': minor
```

Comment:
We typically use `patch` for non-breaking changes for 0.x packages. It doesn't matter too much in this case since canon will already get a minor bump anyway so fine to merge as is